charts/regproc-group2/templates/service.yaml

```diff
@@ -22,7 +22,14 @@ spec:
   loadBalancerIP: {{ .Values.service.loadBalancerIP }}
   {{- end }}
   ports:
-    - port: {{ .Values.service.port }}
-      protocol: TCP 
-      targetPort: {{ .Values.springServicePort }}
+    - name: securezone
```

Comment:
do not hardcode values.

please pass it via values.yaml in range format as used for other services.yaml